### PR TITLE
Docker based setup

### PR DESCRIPTION
**Pull request by dbrgn**
This adds a Docker based setup that can be used both for developing and deployment. No local PHP/Apache installation is required, only Docker.

README is updated with build instructions. Testing welcome.

Replaces #1.

### CODE REVIEW

**Comment by dbrgn** [comment]
If you're happy with this setup, we can configure a CI runner (I like CircleCI) to automatically build the website image on every push. Then, we can pull & deploy it on the server automatically.

**Comment by dbrgn** [comment]
> If you're happy with this setup, we can configure a CI runner (I like CircleCI) to automatically build the website image on every push.

This is totally set up now :slightly_smiling_face: https://hub.docker.com/r/spaceapi/website/tags/

**Comment by dbrgn** [comment]
...and I also set up a deployment on our server and auto-reload with [watchtower](https://github.com/v2tec/watchtower)... Worked beautifully :slightly_smiling_face: 

```
time="2018-08-14T23:22:19Z" level=info msg="Found new spaceapi/website:v1 image (sha256:16da3e5a08f0e5fa71b3a812790d4f0e9ccfc03cb9446d7c9273d0ca275b4beb)"
time="2018-08-14T23:22:19Z" level=info msg="Stopping /website (2531fa52f5f0f171d7b49ec77bc8d061e2f6a1d71f8a7334d1bfe9918801e759) with SIGTERM"
time="2018-08-14T23:22:20Z" level=info msg="Creating /website"
```

Test website (since DNS isn't updated so far): http://any.spaceapi.net:8201/

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            docker build \
+              --no-cache \
+              -t spaceapi/website:latest \
+              -t spaceapi/website:v1 \
+              -t spaceapi/website:${CIRCLE_BRANCH} \
+              .
+      - deploy:
+          name: Push Docker image
+          command: |
+            docker login -u $DOCKER_USER -p $DOCKER_API_KEY && \
```
Comment (rnestler) [inline .circleci/config.yml:21]: I guess these are configured in CircleCI as secret variables, right? Do we need to store the credentials anywhere else, so everybody of our team has access?

File: README.md [hunk 2]
```diff
@@ -1,92 +1,27 @@
-App Kickstarter
-===================
+spaceapi.net Website
+====================
```
Comment (rnestler) [inline README.md]: Nitpick: I prefer the md style headers with `#`.

**Review by rnestler — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dbrgn) [reply README.md]: Changed

File: .circleci/config.yml (hunk 1 above)
Comment (dbrgn) [reply .circleci/config.yml:21]: Yep. These are credentials I created with my own account, but everyone with access to the spaceapi organization on Docker Hub can create new credentials.

**Review by gidsi — APPROVED** [review]

**Pull request merged**